sdk/opendp/smartnoise/_ast/tokens.py

```diff
@@ -129,6 +129,58 @@ def find_nodes(self, type_name, not_child_of=None):
         childnodes = [c.find_nodes(type_name, not_child_of) for c in sqlnodes]
         return nodes + flatten(childnodes)
 
+    def visualize(self, color_types={}, n_trunc=None):
+        """
+        Construct the AST graph of the object
+
+        Args:
+            color_type (optional): A dictionnary which contains the type and
+                the color of the nodes.
+                If no color is provided, the displayed node will be black.
+                Example: {str: 'red', Query: 'green'}
+            n_trunc: for visibility, truncate the expressions.
+                By default, there is no truncation.
+
+        Returns:
+            graphviz Digraph
+
+        """
+        def _label_node(expr, n_trunc):
+            str_expr = str(expr)
+            if n_trunc and len(str_expr) > n_trunc:
+                str_expr = str_expr[:n_trunc] + '...'
+            return f"{type(expr).__name__}: {str_expr}"
+
+        def _color_node(expr, color_types):
+            expr_type = type(expr)
+            if expr_type in color_types.keys():
+                return color_types[expr_type]
+            return 'black'
```

Comment:
Can we make this a constant to explain why we pick this color as the default. I imagine it is the base color for the viz